quickstarts/microsoft.network/front-door-custom-domain/README.md

```diff
@@ -17,14 +17,13 @@
 
 This template Creates a Front Door configuration with a single backend, onboards a custom domain with a path match '/*' for default frontend host and custom domain, and then secures custom domain with a Front Door managed certificate.
 
-Parameters for this template are
-frontDoorName - Name of the frontdoor (ex: contoso)
-customDomainName - FQDN name of the custom domain (ex: www.contoso.com)
-backendaddress - FQDN of the backend (ex: www.contoso-backend.azurewebsites.com)
+Parameters for this template are:
+- `frontDoorName` - Name of the Front Door (ex: contoso)
+- `customDomainName` - FQDN name of the custom domain (ex: www.contoso.com)
```

Comment:
CustomDomainName is not FQDN. Customer can choose any name for the custom domain. For Custom Domains added through portal, the CustomDomainName is generated by replacing dot(.) with hyphen (-) in the FQDN. So if www.contoso.com was added through portal, the custom domain name would be www-contoso-com

Reply:
Thanks @ravindp - I've updated the description and the way the frontend names are created.